src/server/stub.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"log"
+	"net/http"
+	"person"
+)
+
+// Request is a something...
+type Request struct {
+	Obj person.Model `json:"person"`
+}
+
+// Response is a something...
+type Response struct {
+	Error  error        `json:"error"`
+	Object person.Model `json:"object,omitempty"`
```

Comment:
Since the Delete handler never actually used Object and the original "resp" anonymous struct omitted the Obj field, I added ",omitempty" which will omit the field if it's empty -- which in Delete it is.